### PR TITLE
Volo/set up artem vmv

### PR DESCRIPTION
**Pull request by Martyniuk**

### CODE REVIEW

File: artemVMV/init/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Init</title>
```
Comment (Martyniuk) [inline artemVMV/init/index.html:7]: why title is starnge

File: commitlint.config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   extends: ["@commitlint/config-conventional"],
   rules: {
-    "scope-enum": [2, "always", ["artem", "volo", "project"]],
+    "scope-enum": [2, "always", ["artem", "volo", "artemvmv", "project"]],
```
Comment (Martyniuk) [inline commitlint.config.js:4]: one more comment

**Review by Martyniuk — COMMENTED** [review]
Why there is an empty file as CSS.

**Review by ArtemVMV — APPROVED** [review]
Good !)

**Pull request merged**